pheps/phep-0001.md

```diff
@@ -0,0 +1,366 @@
+
+PHEPs may have a Requires header, indicating the PHEP numbers that this PHEP depends on.
+
+PHEPs may also have a Replaced-By header indicating that a PHEP has been rendered obsolete by a later document; the value is the number of the PHEP that replaces the current document. The newer PHEP must have a Replaces header containing the number of the PHEP that it rendered obsolete.
+
+# Auxiliary Files
+<a name="auxiliary-files"></a>
+PHEPs may include auxiliary files such as diagrams. Such files must be placed in a subdirectory called `phep-XXXX`, where "XXXX" is the PHEP number. There are no constraints on the names used in files.
+
+# Transferring PHEP Ownership
+<a name="transferring-phep-ownership"></a>
+It occasionally becomes necessary to transfer ownership of PHEPs to a new author. It is preferable to retain the original author as a co-author of the transferred PHEP, at their discretion. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the PHEP process, or is unreachable. A bad reason to transfer ownership is because the author doesn't agree with the direction of the PHEP. One aim of the PHEP process is to try to build consensus around a PHEP, but if that's not possible, an author can always submit a competing PHEP.
+
+If you are interested in assuming ownership of a PHEP, mention the original author and `@heliophysicsPy/phep-editors` in a comment on the PHEP's pull request. If the original author doesn't respond in a timely manner, the PHEP editors will make a unilateral decision, which can be reversed.
+
+# PHEP Editor Responsibilities & Workflow
+<a name="phep-editor-responsibilities-workflow"></a>
+A PHEP editor must be added to the `@heliophysicsPy/phep-editors` group on GitHub and must watch the `standards` repository.
+
+For each new PHEP that comes in, an editor does the following:
+
+* Read the PHEP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to be accepted.
+
+* The title should accurately describe the content.
+
+* The PHEP's type is correctly labeled ("Standards Track", "Informational", or "Process"), and its status is "Draft".
+
+* The file name extension is correct (i.e. `.md`).
+
+* Skim the PHEP for obvious defects in language (spelling, grammar, sentence structure, etc.) or formatting. Editors may correct problems themselves, but are not required to do so.
+
+* If a project is portrayed as benefiting from or supporting the PHEP, make sure there is some direct indication from the project included to make the support clear.
+
+If the PHEP isn't ready, an editor will send it back to the author for revision, with specific instructions.
```

Comment:
Do we have any kind of requested timeline for revisions?

Reply:
Nope, I think this falls into the category of editorial discretion--there's the provision (up at line 135) for editors to poke the author about a PHEP they're not working on. We could edit that to specifically apply to this state, or assume it applies (except I probably wouldn't have the editors force it to a vote), or just figure if somebody cares enough to write a PHEP maybe they should carry through.

Editorial discretion seems to make the most sense. "Hey, you haven't touched this in three months, one more and we close the PR."